nameko/constants.py

```diff
@@ -3,6 +3,7 @@
 RPC_EXCHANGE_CONFIG_KEY = 'rpc_exchange'
 SERIALIZER_CONFIG_KEY = 'serializer'
 HEARTBEAT_CONFIG_KEY = 'HEARTBEAT'
+SSL_CONFIG_KEY = 'ssl'
```

Comment:
while we aren't exactly in a good place at the moment, i'd like to at least consider a more specific key here. we are currently often conflating rpc and amqp and have e.g. a top level key `heardbeat` which only applies to the rabbit broker, but unlike "heartbeat", ssl might confuse users of the web extension.

whilst fixing existing ones may require breaking backwards compat and is a job for later, i thought this worth bringing up before adding to this